### DISCUSSION TITLE
All my USBs while booting installer are turning off

### DISCUSSION THREAD

**Discussion by lentioszka** [ChefKissInc/NootedRed#310] [General]
### macOS Version

Ventura

### What is your CPU's model?

Ryzen 7 5700G

### Please describe the behaviour in detail.

hi so i wanted to install macos 13.7 on my pc, and i did it but i forgot to add the NootedRed kext file so the graphics acceleration didnt work, after adding the kext file my all usb while booting the installers just turned off, and i tried to modify the config but nothing works, can someone help me plz

my specs:
mb:asus tuf gaming b550 plus
cpu:ryzen 7 5700g
apu:radeon Vega 8

here is my efi
[EFI.zip](https://github.com/user-attachments/files/17356323/EFI.zip)

### What should've happened instead?

it should just work 👍🏻 

### If applicable, attach the .gpuRestart, .panic, etc file related to this issue.

were tf is that

**Comment by hg13bs** [comment]
need to make EFI from scratch without using configurator, read https://chefkissinc.github.io/guides/hackintosh for guide and don't use configurator
